src/toolong/format_parser.py

```diff
@@ -103,7 +103,55 @@ def parse(self, line: str) -> ParseResult | None:
         return timestamp, line, text
 
 
+class EliotLogFormat(LogFormat):
+    """Parser for Eliot log format."""
+    
+    def __init__(self):
+        self._task_cache: Dict[str, Dict[str, Any]] = {}
+    
+    def parse(self, line: str) -> ParseResult | None:
+        try:
+            data = json.loads(line)
+            
+            # Check if this is an Eliot log by looking for required fields
```

Comment:
You can return early and de-indent the rest of the code
```suggestion
            data = json.loads(line)         
        except (json.JSONDecodeError, KeyError):
            return None
            
        # Check if this is an Eliot log by looking for required fields
```